openshift-infra/roles/vpc/tasks/main.yml

```diff
@@ -0,0 +1,136 @@
+---
+
+# roles/vpc/tasks/main.yml
+
+
+# First task : creating the VPC.
+# We are using the variables set in the vars.yml file.
+# The module gives us back its result,
+# which contains information about our new VPC. 
+# We register it in the variable my_vpc.
+
+- name:               Create VPC
+  ec2_vpc_net:
+    name:             "{{ vpc_name }}"
+    cidr_block:       "{{ vpc_cidr_block }}"
+    region:           "{{ aws_region }}"
+    aws_access_key:   "{{ ec2_access_key }}"
+    aws_secret_key:   "{{ ec2_secret_key }}"
```

Comment:
Yes.  When you access his environment for the first time, you'll want to go to IAM | Users | find your user, create a new access key / secret. Make the old one active.